data/campaigns/Northern_Rebirth/scenarios/14a_Epilogue.cfg

```diff
@@ -408,7 +408,7 @@
                 [/message]
                 [message]
                     speaker=narrator
-                    message= _ "On Father Morvin’s advice, the Council approached all the different orcish tribes and made treaties with them. If a chieftain refused to cooperate with the Alliance, he was forcibly removed from his post and a more favorably-disposed leader put in his place."
+                    message= _ "On Father Morvin’s advice, the Council approached all the different orcish tribes and made treaties with them. If a chieftain refused to cooperate with the Alliance, he was removed and a more favorably-disposed leader put in his place."
```

Comment:
I prefer the original, no reason to be ambiguous about how it works.

Reply:
Anyone else want to weigh in?